### DISCUSSION TITLE
Generation of webapi, inconsistent project names - intentional?

### DISCUSSION THREAD

**Discussion by Tungsten78** [nx-dotnet/nx-dotnet#605] [General]
![image](https://user-images.githubusercontent.com/3805338/215279625-50a21478-38bf-4371-8144-d08582b90df1.png)
![image](https://user-images.githubusercontent.com/3805338/215279478-869b26c0-73d0-40ca-a266-d8f9f4045189.png)

1. should the project names be consistent?
2. should the generated projects have honored the directory option

**Comment by AgentEnder** [comment]
It wouldn't be a bad idea for the generated projects to not contain the generated prefix, although folks shouldn't really be manually running commands against them so I wouldn't consider it high priority in terms of changes to make...

RE: directory option, similarly, folks shouldn't really be looking through the directory structure under generated very often... but I can see cases where they may in terms of jumping to the files with the TS language server... we could definitely obey the directory option, we would just need to pass it through in the project generator into the swagger generator, and then into the nrwl/js generator.

**Reply by Tungsten78** [reply]
It was more that the two generated projects were inconsistent. I would vote for omitting generated in the project name.
